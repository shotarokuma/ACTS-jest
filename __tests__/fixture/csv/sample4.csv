Column1;Column2;Column3;Column4;Column5;Column6;Column7;Column8;Column9;Column10;Column11;Column12;Column13;Column14;Column15;Column16;Column17;Column18;Column19;Column20
sample cell,;true;sample cell,;false;true;false;true;false;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;false;true;sample cell,;sample cell,;sample cell,;sample cell,;false
sample cell,;true;true;false;sample cell,;sample cell,;false;sample cell,;false;true;sample cell,;true;sample cell,;true;sample cell,;true;sample cell,;true;sample cell,;sample cell,
true;false;true;false;true;true;true;sample cell,;false;sample cell,;sample cell,;sample cell,;sample cell,;false;false;sample cell,;sample cell,;sample cell,;false;sample cell,
true;sample cell,;sample cell,;sample cell,;true;true;true;sample cell,;sample cell,;sample cell,;false;false;false;false;sample cell,;sample cell,;false;false;false;false
false;sample cell,;sample cell,;false;true;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;true;false;sample cell,;false;true;sample cell,;sample cell,;false;false
sample cell,;sample cell,;true;false;false;true;sample cell,;false;false;sample cell,;true;sample cell,;true;false;true;sample cell,;sample cell,;true;false;sample cell,
sample cell,;sample cell,;sample cell,;true;true;true;sample cell,;true;sample cell,;sample cell,;sample cell,;sample cell,;true;false;false;sample cell,;false;true;sample cell,;sample cell,
sample cell,;sample cell,;sample cell,;false;sample cell,;true;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;true;sample cell,;sample cell,;false;true;sample cell,;sample cell,;sample cell,
true;false;sample cell,;sample cell,;sample cell,;false;false;true;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;true;sample cell,;sample cell,;false;true;false;true
sample cell,;sample cell,;sample cell,;false;false;false;sample cell,;sample cell,;sample cell,;sample cell,;false;sample cell,;sample cell,;sample cell,;false;false;sample cell,;true;true;sample cell,
sample cell,;true;sample cell,;sample cell,;false;sample cell,;sample cell,;sample cell,;false;true;true;true;sample cell,;true;false;sample cell,;false;sample cell,;false;sample cell,
false;true;sample cell,;true;sample cell,;sample cell,;sample cell,;sample cell,;true;false;true;sample cell,;true;false;true;false;sample cell,;false;sample cell,;sample cell,
sample cell,;sample cell,;false;sample cell,;sample cell,;false;sample cell,;true;sample cell,;sample cell,;false;sample cell,;false;sample cell,;true;sample cell,;sample cell,;false;false;sample cell,
true;sample cell,;false;false;sample cell,;sample cell,;sample cell,;true;sample cell,;sample cell,;true;sample cell,;false;true;true;true;false;sample cell,;sample cell,;false
sample cell,;sample cell,;false;sample cell,;true;sample cell,;sample cell,;true;true;true;sample cell,;true;sample cell,;true;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,
sample cell,;sample cell,;sample cell,;sample cell,;false;sample cell,;false;false;sample cell,;false;false;sample cell,;false;sample cell,;sample cell,;false;sample cell,;false;sample cell,;sample cell,
false;true;false;true;false;true;sample cell,;sample cell,;false;sample cell,;sample cell,;sample cell,;sample cell,;true;true;true;sample cell,;sample cell,;false;false
sample cell,;sample cell,;true;true;true;false;sample cell,;sample cell,;sample cell,;true;true;sample cell,;true;sample cell,;false;true;sample cell,;sample cell,;sample cell,;sample cell,
sample cell,;sample cell,;sample cell,;sample cell,;false;false;false;false;sample cell,;sample cell,;true;false;true;sample cell,;sample cell,;sample cell,;sample cell,;false;sample cell,;false
false;sample cell,;sample cell,;true;true;sample cell,;true;true;sample cell,;true;false;sample cell,;sample cell,;false;sample cell,;false;sample cell,;false;false;true
false;false;true;false;true;true;true;false;sample cell,;true;false;true;sample cell,;sample cell,;true;true;true;false;true;sample cell,
sample cell,;true;sample cell,;true;sample cell,;sample cell,;sample cell,;sample cell,;true;false;true;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;false;true;true;true
sample cell,;sample cell,;false;sample cell,;sample cell,;true;sample cell,;false;sample cell,;false;true;sample cell,;false;sample cell,;sample cell,;true;true;false;sample cell,;true
false;sample cell,;true;sample cell,;sample cell,;false;true;sample cell,;false;true;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;false;false;false;true;false
false;sample cell,;false;sample cell,;sample cell,;true;sample cell,;true;sample cell,;sample cell,;sample cell,;sample cell,;false;sample cell,;sample cell,;sample cell,;sample cell,;false;true;false
sample cell,;sample cell,;true;false;false;sample cell,;false;sample cell,;true;false;sample cell,;sample cell,;sample cell,;sample cell,;sample cell,;true;true;sample cell,;false;sample cell,
false;sample cell,;true;sample cell,;sample cell,;sample cell,;true;false;false;sample cell,;sample cell,;false;false;false;true;false;false;sample cell,;false;false
sample cell,;false;true;sample cell,;true;false;sample cell,;false;false;sample cell,;false;sample cell,;sample cell,;sample cell,;false;sample cell,;false;sample cell,;true;false
false;sample cell,;sample cell,;true;sample cell,;false;false;sample cell,;false;false;sample cell,;true;sample cell,;false;true;true;true;true;sample cell,;true
sample cell,;sample cell,;true;false;true;sample cell,;true;sample cell,;true;false;false;true;sample cell,;sample cell,;false;sample cell,;false;false;sample cell,;true
