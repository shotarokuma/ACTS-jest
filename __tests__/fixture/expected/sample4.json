[
{"Column1":"sample cell,","Column2":true,"Column3":"sample cell,","Column4":false,"Column5":true,"Column6":false,"Column7":true,"Column8":false,"Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":false,"Column15":true,"Column16":"sample cell,","Column17":"sample cell,","Column18":"sample cell,","Column19":"sample cell,","Column20":false},
{"Column1":"sample cell,","Column2":true,"Column3":true,"Column4":false,"Column5":"sample cell,","Column6":"sample cell,","Column7":false,"Column8":"sample cell,","Column9":false,"Column10":true,"Column11":"sample cell,","Column12":true,"Column13":"sample cell,","Column14":true,"Column15":"sample cell,","Column16":true,"Column17":"sample cell,","Column18":true,"Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":true,"Column2":false,"Column3":true,"Column4":false,"Column5":true,"Column6":true,"Column7":true,"Column8":"sample cell,","Column9":false,"Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":false,"Column15":false,"Column16":"sample cell,","Column17":"sample cell,","Column18":"sample cell,","Column19":false,"Column20":"sample cell,"},
{"Column1":true,"Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":true,"Column6":true,"Column7":true,"Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":false,"Column12":false,"Column13":false,"Column14":false,"Column15":"sample cell,","Column16":"sample cell,","Column17":false,"Column18":false,"Column19":false,"Column20":false},
{"Column1":false,"Column2":"sample cell,","Column3":"sample cell,","Column4":false,"Column5":true,"Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":true,"Column13":false,"Column14":"sample cell,","Column15":false,"Column16":true,"Column17":"sample cell,","Column18":"sample cell,","Column19":false,"Column20":false},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":true,"Column4":false,"Column5":false,"Column6":true,"Column7":"sample cell,","Column8":false,"Column9":false,"Column10":"sample cell,","Column11":true,"Column12":"sample cell,","Column13":true,"Column14":false,"Column15":true,"Column16":"sample cell,","Column17":"sample cell,","Column18":true,"Column19":false,"Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":true,"Column5":true,"Column6":true,"Column7":"sample cell,","Column8":true,"Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":true,"Column14":false,"Column15":false,"Column16":"sample cell,","Column17":false,"Column18":true,"Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":false,"Column5":"sample cell,","Column6":true,"Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":true,"Column14":"sample cell,","Column15":"sample cell,","Column16":false,"Column17":true,"Column18":"sample cell,","Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":true,"Column2":false,"Column3":"sample cell,","Column4":"sample cell,","Column5":"sample cell,","Column6":false,"Column7":false,"Column8":true,"Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":true,"Column15":"sample cell,","Column16":"sample cell,","Column17":false,"Column18":true,"Column19":false,"Column20":true},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":false,"Column5":false,"Column6":false,"Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":false,"Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":false,"Column16":false,"Column17":"sample cell,","Column18":true,"Column19":true,"Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":true,"Column3":"sample cell,","Column4":"sample cell,","Column5":false,"Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":false,"Column10":true,"Column11":true,"Column12":true,"Column13":"sample cell,","Column14":true,"Column15":false,"Column16":"sample cell,","Column17":false,"Column18":"sample cell,","Column19":false,"Column20":"sample cell,"},
{"Column1":false,"Column2":true,"Column3":"sample cell,","Column4":true,"Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":true,"Column10":false,"Column11":true,"Column12":"sample cell,","Column13":true,"Column14":false,"Column15":true,"Column16":false,"Column17":"sample cell,","Column18":false,"Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":false,"Column4":"sample cell,","Column5":"sample cell,","Column6":false,"Column7":"sample cell,","Column8":true,"Column9":"sample cell,","Column10":"sample cell,","Column11":false,"Column12":"sample cell,","Column13":false,"Column14":"sample cell,","Column15":true,"Column16":"sample cell,","Column17":"sample cell,","Column18":false,"Column19":false,"Column20":"sample cell,"},
{"Column1":true,"Column2":"sample cell,","Column3":false,"Column4":false,"Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":true,"Column9":"sample cell,","Column10":"sample cell,","Column11":true,"Column12":"sample cell,","Column13":false,"Column14":true,"Column15":true,"Column16":true,"Column17":false,"Column18":"sample cell,","Column19":"sample cell,","Column20":false},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":false,"Column4":"sample cell,","Column5":true,"Column6":"sample cell,","Column7":"sample cell,","Column8":true,"Column9":true,"Column10":true,"Column11":"sample cell,","Column12":true,"Column13":"sample cell,","Column14":true,"Column15":"sample cell,","Column16":"sample cell,","Column17":"sample cell,","Column18":"sample cell,","Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":false,"Column6":"sample cell,","Column7":false,"Column8":false,"Column9":"sample cell,","Column10":false,"Column11":false,"Column12":"sample cell,","Column13":false,"Column14":"sample cell,","Column15":"sample cell,","Column16":false,"Column17":"sample cell,","Column18":false,"Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":false,"Column2":true,"Column3":false,"Column4":true,"Column5":false,"Column6":true,"Column7":"sample cell,","Column8":"sample cell,","Column9":false,"Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":true,"Column15":true,"Column16":true,"Column17":"sample cell,","Column18":"sample cell,","Column19":false,"Column20":false},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":true,"Column4":true,"Column5":true,"Column6":false,"Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":true,"Column11":true,"Column12":"sample cell,","Column13":true,"Column14":"sample cell,","Column15":false,"Column16":true,"Column17":"sample cell,","Column18":"sample cell,","Column19":"sample cell,","Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":false,"Column6":false,"Column7":false,"Column8":false,"Column9":"sample cell,","Column10":"sample cell,","Column11":true,"Column12":false,"Column13":true,"Column14":"sample cell,","Column15":"sample cell,","Column16":"sample cell,","Column17":"sample cell,","Column18":false,"Column19":"sample cell,","Column20":false},
{"Column1":false,"Column2":"sample cell,","Column3":"sample cell,","Column4":true,"Column5":true,"Column6":"sample cell,","Column7":true,"Column8":true,"Column9":"sample cell,","Column10":true,"Column11":false,"Column12":"sample cell,","Column13":"sample cell,","Column14":false,"Column15":"sample cell,","Column16":false,"Column17":"sample cell,","Column18":false,"Column19":false,"Column20":true},
{"Column1":false,"Column2":false,"Column3":true,"Column4":false,"Column5":true,"Column6":true,"Column7":true,"Column8":false,"Column9":"sample cell,","Column10":true,"Column11":false,"Column12":true,"Column13":"sample cell,","Column14":"sample cell,","Column15":true,"Column16":true,"Column17":true,"Column18":false,"Column19":true,"Column20":"sample cell,"},
{"Column1":"sample cell,","Column2":true,"Column3":"sample cell,","Column4":true,"Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":true,"Column10":false,"Column11":true,"Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":"sample cell,","Column16":"sample cell,","Column17":false,"Column18":true,"Column19":true,"Column20":true},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":false,"Column4":"sample cell,","Column5":"sample cell,","Column6":true,"Column7":"sample cell,","Column8":false,"Column9":"sample cell,","Column10":false,"Column11":true,"Column12":"sample cell,","Column13":false,"Column14":"sample cell,","Column15":"sample cell,","Column16":true,"Column17":true,"Column18":false,"Column19":"sample cell,","Column20":true},
{"Column1":false,"Column2":"sample cell,","Column3":true,"Column4":"sample cell,","Column5":"sample cell,","Column6":false,"Column7":true,"Column8":"sample cell,","Column9":false,"Column10":true,"Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":"sample cell,","Column16":false,"Column17":false,"Column18":false,"Column19":true,"Column20":false},
{"Column1":false,"Column2":"sample cell,","Column3":false,"Column4":"sample cell,","Column5":"sample cell,","Column6":true,"Column7":"sample cell,","Column8":true,"Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":false,"Column14":"sample cell,","Column15":"sample cell,","Column16":"sample cell,","Column17":"sample cell,","Column18":false,"Column19":true,"Column20":false},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":true,"Column4":false,"Column5":false,"Column6":"sample cell,","Column7":false,"Column8":"sample cell,","Column9":true,"Column10":false,"Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":"sample cell,","Column16":true,"Column17":true,"Column18":"sample cell,","Column19":false,"Column20":"sample cell,"},
{"Column1":false,"Column2":"sample cell,","Column3":true,"Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":true,"Column8":false,"Column9":false,"Column10":"sample cell,","Column11":"sample cell,","Column12":false,"Column13":false,"Column14":false,"Column15":true,"Column16":false,"Column17":false,"Column18":"sample cell,","Column19":false,"Column20":false},
{"Column1":"sample cell,","Column2":false,"Column3":true,"Column4":"sample cell,","Column5":true,"Column6":false,"Column7":"sample cell,","Column8":false,"Column9":false,"Column10":"sample cell,","Column11":false,"Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":false,"Column16":"sample cell,","Column17":false,"Column18":"sample cell,","Column19":true,"Column20":false},
{"Column1":false,"Column2":"sample cell,","Column3":"sample cell,","Column4":true,"Column5":"sample cell,","Column6":false,"Column7":false,"Column8":"sample cell,","Column9":false,"Column10":false,"Column11":"sample cell,","Column12":true,"Column13":"sample cell,","Column14":false,"Column15":true,"Column16":true,"Column17":true,"Column18":true,"Column19":"sample cell,","Column20":true},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":true,"Column4":false,"Column5":true,"Column6":"sample cell,","Column7":true,"Column8":"sample cell,","Column9":true,"Column10":false,"Column11":false,"Column12":true,"Column13":"sample cell,","Column14":"sample cell,","Column15":false,"Column16":"sample cell,","Column17":false,"Column18":false,"Column19":"sample cell,","Column20":true}
]
